etcdserver/raft.go

```diff
@@ -97,6 +97,7 @@ type raftNode struct {
 	term         uint64
 	lead         uint64
 
+	tickMu *sync.Mutex
```

Comment:
Out of curiosity, any reason for pointer here? My understanding is that pointers are not typically needed for sync.Mutex.

Reply:
I'm planning to backport this to 3.1 without a pointer since that removes the need to initialize the mutex, which simplifies the backport: https://github.com/coreos/etcd/pull/9500/files#diff-8c6a0ae3bb0763acd9c96a35d89131feR99

Reply:
@jpbetz govet would complain something like this

> passes lock by value: github.com/coreos/etcd/clientv3.Client contains sync.Mutex